config/application.rb

```diff
@@ -28,5 +28,7 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+
+    # ActiveSupport::Deprecation.silenced = true
```

Comment:
I don't think this need to be in here - its commented out and wont be having any effect